PowerEditor/installer/nativeLang/polish.xml

```diff
@@ -104,9 +104,11 @@ Translation note:
 					<Item id="41003" name="Za&amp;mknij"/>
 					<Item id="41004" name="Zamknij &amp;wszystko"/>
 					<Item id="41005" name="Zamknij wszystko prócz bieżącego pliku"/>
+					<Item CMDID="41026" name="Zamknij wszystko oprócz przypiętych"/>
 					<Item id="41009" name="Zamknij wszystko na lewo od bieżącego pliku"/>
 					<Item id="41018" name="Zamknij wszystko na prawo od bieżącego pliku"/>
 					<Item id="41024" name="Zamknij wszystko, gdzie nie dokonano zmian"/>
+					<Item CMDID="44048" name="Przypnij kartę" alternativeName="Odepnij kartę"/>
```

Comment:
It should be "id" instead of "CMDID".

Reply:
I don't see what else isn't translated to be honest

Reply:
<Item id="44048" name="Przypnij kartę" alternativeName="Odepnij kartę"/>
instead of
<Item CMDID="44048" name="Przypnij kartę" alternativeName="Odepnij kartę"/>